feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go

```diff
@@ -37,7 +37,7 @@ func TestMain(m *testing.M) {
 
 const (
 	PTISIS   = telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_ISIS
-	ISISName = "osiris"
+	ISISName = "DEFAULT"
```

Comment:
Remove the constant from this file and use `session.ISISName` where necessary.

Reply:
updated. thank you